client/webserver/site/src/js/charts.js

```diff
@@ -721,11 +721,19 @@ function makeLabels (ctx, screenW, min, max, spacingGuess, step, unit) {
   const tick = tickGuess + step - (tickGuess % step)
   let x = min + tick - (min % tick)
   const absMax = Math.max(Math.abs(max), Math.abs(min))
+  const pts = []
+  let widest = 0
+  // if absMax < 1 we can ignore the changes, as it will not have enough
+  // precision for recalculating it.
+  if (absMax < 1) {
+    return {
+      widest: widest,
+      lbls: pts
+    }
+  }
   // The Math.round part is the minimum precision required to see the change in the numbers.
   // The 2 accounts for the precision of the tick.
   const sigFigs = Math.round(Math.log10(absMax / tick)) + 2
```

Comment:
This is almost certainly where the bug is. The intention is to make sure to show enough sig figs that two consecutive ticks marks never show the same value because of rounding. Not sure how this would ever give `sigFigs` equal to zero, since presumably `absMax > 0` and `tick < absMax`.